light-client/src/fixture/localnet.rs

```diff
@@ -0,0 +1,150 @@
+use hex_literal::hex;
+
+use std::prelude::rust_2015::Vec;
+
+use crate::header::eth_header::ETHHeader;
+use crate::header::eth_headers::ETHHeaders;
+
+use crate::fixture::{decode_header, Network, UpdateClientEpochInput, UpdateClientNonEpochInput};
+use crate::misc::{Address, ChainId};
+
+pub struct Localnet;
+
+impl Network for Localnet {
```

Comment:
Test data can be prepared for each network for future HF support. Currently, Bohr is only operating on the local net, so the test data is for the local net only.
We will increase the corresponding test data once it is deployed on the test net and main net.